tools/bumbershoot/idpassemble/.shed.yml

```diff
@@ -0,0 +1,6 @@
+# repository published to https://toolshed.g2.bx.psu.edu/repos/galaxyp/idpqonvert
+categories:
+- Proteomics
```

Comment:
Can you please add a long_descriptions and shorten the description. Also please add repository_url.